bot/resources/tags/json.md

```diff
@@ -0,0 +1,15 @@
+JSON is a very handy way of storing values. But when starting out with JSON, you might run into the following error.
```

Comment:
> But when starting out with JSON

This statement could be interpreted as implying the error is a beginner issue, but that isn't really relevant. The emphasis should be on the fact that this often happens when a new empty file is created/opened, not on anything relating to the user's level of experience.